src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -788,9 +788,20 @@ public void onEmojiSelected(String emoji) throws InvalidMessageException {
         //resets the local view to render new reaction
         fragment.getListAdapter().notifyDataSetChanged();
 
+        String myAddress = "";
+        IdentityDatabase identityDatabase = DatabaseFactory.getIdentityDatabase(getApplicationContext());
+        try {
+          myAddress = identityDatabase.getMyIdentity().getAddress().serialize();
+        } catch (Exception e) {
+          e.printStackTrace();
+        }
+
         String body = "{\"type\": \"reaction\", \"hash\": \"" + messageRecord.getHash() + "\", \"emoji\": \"" +
```

Comment:
Think this line could get much clearer if we were to use some JSON serialization from a map

Reply:
yess we could have a builder class or something that sets all the attrs

Reply:
Pushed the change